.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: ci
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version:
+          - 8
+          - 9
+          - 10
+          - 11
+          - 12
+          - 13
```

Comment:
We don't need to test versions other than 8, 11 and 17.

Reply:
"Migrate" shouldn't change test coverage.